packaging/ubuntu-14.04/rules

```diff
@@ -54,7 +54,11 @@ endif
 # security of the system.  Discuss a proper approach to this for downstreams
 # if and when they approach us
 ifeq ($(shell dpkg-vendor --query Vendor),Ubuntu)
-    VENDOR_ARGS=--enable-nvidia-ubuntu
+	# On Ubuntu 14.04 snapd cannot add the libcap dependency because of a
```

Comment:
I don't understand the indentation here, it doesn't seem to align with the after the comment block